datafusion/sqllogictest/test_files/ddl.slt

```diff
@@ -828,3 +828,39 @@ drop table table_with_pk;
 statement ok
 set datafusion.catalog.information_schema = false;
 
+
+##########
+# Normalize column names in constraints
```

Comment:
Another potential location for this kind of test would be https://github.com/apache/datafusion/blob/main/datafusion/sqllogictest/test_files/ident_normalization.slt

But keeping it with DDL also makes sense too

Reply:
Moved. Thanks @alamb for the review.